combat-trainer.lic

```diff
@@ -2032,6 +2038,19 @@ class TrainerProcess
 
   private
 
+  def use_almanac(game_state)
+    return unless Time.now - UserVars.almanac_last_use >= 600
+    return if left_hand && !game_state.currently_whirlwinding
+
+    game_state.sheath_whirlwind_offhand if game_state.currently_whirlwinding
+    bput('get my almanac', 'You get', 'What were')
+    bput('study my almanac', 'You set about', 'gleaned all the insight you can', 'Study what')
+    waitrt?
```

Comment:
❓ Since there's roundtime, should we retreat first?

Reply:
added a retreat and engage